content/participants/ttjc@Megha-Pathak/index.md

```diff
@@ -39,20 +39,40 @@ I have done projects along side with learning technologies some of them are:
 
 My projects:
 
-#### Not Hacked: Card Game to educate kids about Cyber Security
+#### LadeBug: Meet your online digital Ladybug for companionship & debugging -- LadeBug!
 
-_description_ This is a card game that takes the player through different privacy situations and ask for the response they will choose. The points given to them specifies how good their choice is. 
 
-_stack_ Made with HTML, CSS and JavaScript.
+_description_ A chrome extension to overlay a tiny ladybug, as a digital companion during debugging and general development work. Follow the Set-Up Instructions present on the repository.  
 
-_github link_ https://github.com/Megha-Pathak/Not-Hacked
+_stack_ Made with JavaScript.
 
-#### Know More: A Twitter Bot
+_github link_ https://github.com/Megha-Pathak/LadeBug
```

Comment:
hosted link is missing 🤔🤔!!

Reply:
Hey @gkanishk 

That project is a Chrome Extension. For using it, a user needs to `Load Unpacked`. I have included this in the Installation instruction. 
The only way to host the extension is to publish it on Chrome Store. I tried doing it, but there is some issue with the promotional image. As it has nothing to do with the development and usage of the extension, I suggested the mentors to directly set it up using the installation instructions. 
One of the mentor agreed that this is not an issue.

Reply:
But still the extension should be live and available on Chrome store for everyone to use, I would suggest to publish it or tell me if I could help with the issue.

Reply:
Have you tried the fix?

Reply:
Hey @gkanishk 
I talked to some extension experts. They too were not able to figure out the issue with the chrome web store. One of them suggested an alternative for the extension distribution, so I worked on it. 

I have updated my PR with the needed the hosting link. I believe we are good to go from here.

Reply:
@Megha-Pathak So you are not publishing it!! 😂😂😂

Reply:
I didn't say that @gkanishk 
If I publish it in upcoming weeks, I will edit the link in the landing website of extension itself.